recipes/sphinx_removed_in/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "sphinx_removed_in" %}
```

```suggestion
{% set name = "sphinx-removed-in" %}
```

I would follow the pypi name